neverbounce_sdk/utils.py

```diff
@@ -12,7 +12,7 @@
 API_VERSION = 'v4'
 
 
-def urlfor(*parts):
+def urlfor(*parts, api_version=API_VERSION):
```

Comment:
We won't be able to use this syntax yet, some internal applications still use Python `2.7`. We may need to pass the version as the first`*parts` argument instead. 

For example instead of calling `urlfor` like this:
```
urlfor('poe', 'confirm', api_version=self.api_version)
```
We could do:
```
urlfor(self.api_version, 'poe', 'confirm')
```

Reply:
Please check again. I kept `urlfor` for backward compatibility.